parser-typechecker/src/Unison/Codebase/SqliteCodebase.hs

```diff
@@ -782,47 +792,50 @@ sqliteCodebase debugName root localOrRemote action = do
                   $ Ops.lca (Cv.causalHash1to2 h1) (Cv.causalHash1to2 h2) c1 c2
     let codebase =
           C.Codebase
-            (Cache.applyDefined termCache getTerm)
-            (Cache.applyDefined typeOfTermCache getTypeOfTermImpl)
-            (Cache.applyDefined declCache getTypeDeclaration)
-            putTerm
-            putTypeDeclaration
-            -- _getTermComponent
-            getTermComponentWithTypes
-            getDeclComponent
-            getCycleLength
-            (getRootBranch rootBranchCache)
-            getRootBranchExists
-            (putRootBranch rootBranchCache)
-            (rootBranchUpdates rootBranchCache)
-            getBranchForHash
-            putBranch
-            isCausalHash
-            getPatch
-            putPatch
-            patchExists
-            dependentsImpl
-            dependentsOfComponentImpl
-            syncFromDirectory
-            syncToDirectory
-            viewRemoteBranch'
-            (\r opts action -> pushGitBranch conn r opts action)
-            watches
-            getWatch
-            putWatch
-            clearWatches
-            getReflog
-            appendReflog
-            termsOfTypeImpl
-            termsMentioningTypeImpl
-            hashLength
-            termReferencesByPrefix
-            declReferencesByPrefix
-            referentsByPrefix
-            branchHashLength
-            branchHashesByPrefix
-            (Just sqlLca)
-            (Just \l r -> runDB conn $ fromJust <$> before l r)
+            { getTerm = (Cache.applyDefined termCache getTerm),
```

Comment:
Naming the fields gives us _much_ better error messages when forgetting to add the new field.